sdk/src/permission.ts

```diff
@@ -6,6 +7,18 @@ export enum Operation {
   GrantWithCondition = 2,
 }
 
+export const ADDRESS_ZERO = getAddress(`0x${'0'.repeat(40)}`); // address(0)
+export const ADDRESS_ONE = getAddress(`0x${'0'.repeat(39)}1`); // address(1)
+export const ADDRESS_TWO = getAddress(`0x${'0'.repeat(39)}2`); // address(2)
+export const ADDRESS_LAST = getAddress(`0x${'f'.repeat(40)}`); // address(-1)
```

Comment:
do we need to export the addresses?
perhaps we should export them as flags not ass address_one ...etc 
i mean the naming convention that used here

Reply:
Fixed in 9c3fae24e676e47cc7e10eee7ba9c7e2030d8266